src/app/FormCreate.jsx

```diff
@@ -59,29 +66,27 @@ export default class FormCreate extends Component {
   }
 
   showPreview() {
-
+    const { activeForm, forms, elkhornHost } = this.props;
     // First test the form is reachable
-    const form = this.props.activeForm ? this.props.forms[this.props.activeForm] : this.props.forms.form;
-    const previewForm = {...form};
-    previewForm.steps[0].widgets = this.props.forms.widgets;
-    const url = `${this.props.app.elkhornHost}/preview.js?props=${encodeURIComponent(JSON.stringify(previewForm))}`;
+    const form = activeForm ? forms[activeForm] : forms.form;
+    const previewForm = { ...form };
+    previewForm.steps[0].widgets = forms.widgets;
+    const url = `${elkhornHost}/preview.js?props=${encodeURIComponent(JSON.stringify(previewForm))}`;
 
     let formCreate = this;
```

Comment:
I love the idea of using a variable with the same name as the file to refer to the 'this' scope. Makes accessing it's properties very clear to read.